src/test/regress/specs/isolation_add_node_vs_copy.spec

```diff
@@ -0,0 +1,72 @@
+# remove one of the nodes for the purpose of the test
+setup
+{	
+	SELECT master_remove_node('localhost', 57638);
+	
+    CREATE TABLE test_reference_table (test_id integer);
+    SELECT create_reference_table('test_reference_table');
+}
+
+# ensure that both nodes exists for the remaining of the isolation tests
+teardown
+{
+	DROP TABLE test_reference_table;
+	SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57637);
+	SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57638);
```

Comment:
If it was independent of place then it shouldn't have to remove a node at the start. That seems like a strange pattern.

To keep things simple, I suggest having 2 types of tests: tests that call master_add_node/master_remove_node and tests that don't. The former type should be before `isolation_cluster_management`, start with an empty `pg_dist_node`, and are not parallelisable. The latter type should be after `isolation_cluster_management` and are mostly parallelisable.